kobuki_driver/CMakeLists.txt

```diff
@@ -27,6 +29,10 @@ find_package(ecl_time REQUIRED)
 ament_package_xml()
 ecl_enable_cxx14_compiler()
 
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
```

Comment:
This is covered in `ecl_enable_cxx_warnings()`.

Reply:
Changed to use `ecl_enable_cxx_warnings()` in 3c23381.